fsutils/mkgpt/mkgpt.c

```diff
@@ -128,12 +129,12 @@ static const uint8_t g_partition_type_swap[16] =
 
 static void get_uuid(FAR uint8_t *uuid)
 {
-  int fd;
-  fd = open("/dev/urandom", O_RDONLY);
-  if (fd > 0)
+  /* call getrandom to read  /dev/urandom */
+
+  if (getrandom(uuid, 16, GRND_NONBLOCK) < 0)
```

Comment:
```suggestion
  if (getrandom(uuid, 16, 0) < 0)
```

Reply:
Yes. This gets back to original behaviour